.github/workflows/create-dev-tag.yml

```diff
@@ -0,0 +1,71 @@
+name: Create Development Tag
+
+on:
+  workflow_dispatch:
+
+  schedule:
+    - cron: '5 20 * * *'
+
+env:
+  GOWORK: off
+  GOPRIVATE: github.com/mesosphere
+  DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+  DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+  GH_ACCESS_TOKEN: ${{ secrets.MESOSPHERECI_USER_TOKEN }}
+
+jobs:
+  create-dev-tag:
+    runs-on:
+    - self-hosted
+    - small
+    strategy:
+      matrix:
+        branch:
+          - main
+          - release-2.1
+          - release-2.2
+          - release-2.3
+          - release-2.4
+          - release-2.5
+          - release-2.6
```

Comment:
We need to automate adding to this list or find a different solution

Reply:
cant we edit the yaml file when a new branch is needed?

Reply:
I mean, we can but we don't want to 🙂